phpstan.neon

```diff
@@ -30,9 +30,16 @@ parameters:
 
     ignoreErrors:
         - '#Class "Rector\\Nette\\Rector\\Class_\\MoveFinalGetUserToCheckRequirementsClassMethodRector" is missing @see annotation with test case class reference#'
+        - '#Class "Rector\\Nette\\Rector\\Class_\\CreateOrUpdateFormDataRector" is missing @see annotation with test case class reference#'
 
         - '#Cognitive complexity for "Rector\\Nette\\Kdyby\\ContributeEventClassResolver\:\:resolveGetterMethodByEventClassAndParam\(\)" is 10, keep it under 9#'
 
+        - '#Cognitive complexity for "Rector\\Nette\\NodeFinder\\FormFinder\:\:findFormVariable\(\)" is 10, keep it under 9#'
+
+        - '#Cognitive complexity for "Rector\\Nette\\NodeFinder\\FormFinder\:\:findFormFields\(\)" is 17, keep it under 9#'
+
+        - '#Cognitive complexity for "Rector\\Nette\\NodeFinder\\FormFinder\:\:findOnSuccessCallback\(\)" is 14, keep it under 9#'
```

Comment:
These should be refactored isntead. 17 is too heavy for non-traversing methods

Reply:
what do you mean by "non-traversing methods"? This finder methods are exactly full of traversing php parser nodes imho:

```
if (! $stmt instanceof Expression) {
    continue;
}
if (! $stmt->expr instanceof Assign) {
    continue;
}

if (! $stmt->expr->var instanceof ArrayDimFetch) {
    continue;
}
```
I would be very happy if there is some generic way how to find nodes in AST

Reply:
I am on 10 now... maybe you'll give me some hints later :)